src/main/protobuf/RoutingCardId.proto

```diff
@@ -4,7 +4,9 @@ import "validate/validate.proto";
 import "scalapb/scalapb.proto";
 import "scalapb/validate.proto";
 
-option java_outer_classname = "RoutingCardIdProto";
+package foo;
+
+option java_outer_classname = "RoutingCardIdProtoX";  // conflicts with FlatPackage without the X suffix
```

Comment:
This was set to the exact name used by the generated Scala code which leads to a name clash between Java and Scala.